### DISCUSSION TITLE
Strange behaviour after adding print level

### DISCUSSION THREAD

**Discussion by gregoriomarchesini** [casadi/casadi#3377] [General]
Dear Casadi Team

I am using opti stack with casadi on Python 3.8 by calling 
```
opti = ca.Opti(''conic')
...
...
opti.solver("qpoases")
```

Then I wanted to silence the output and I added the configuration `printLevel` as

```
opti.solver("qpoases",{"printLevel":0})
```

Doing this changes the behaviour of the solver. Indeed, the solver does not find a solution anymore to my problem by just changing this parameter. I feel this could be like a bug because after I remove the option `printLevel` everything goes back again to normal and a solution is found. Any thoughts

**Comment by jgillis** [comment]
That sounds like a bug. Do share the problem, e.g. by sharing files produced from "dump_in" True option added next to printLevel.

**Reply by gregoriomarchesini** [reply]
I tried to make a very simple test here 

```
import casadi as ca
opti = ca.Opti('conic')
opti.solver("qpoases",{"printLevel":0})
x = opti.variable(1)

cost = x**2
consraint = x**2<=3
opti.minimize(cost)
opti.subject_to(consraint)

opti.solve()
```

I get this error

```
RuntimeError: Error in Opti::solve [OptiNode] at .../casadi/core/optistack.cpp:157:
.../casadi/core/options.cpp:248: Assertion "op.second.can_cast_to(entry->type)" failed:
Illegal type for printLevel: OT_INT cannot be cast to OT_STRING.
```

This suggests that I should put a string for printLevel. Although I tried '0'. 'False', 'None', but none of them worked
Maybe I am doing something stupid

**Reply by gregoriomarchesini** [reply]
Also If I try 

```
opti.solver("qpoases",{"printLevel":"0"})
```

I get 
```
RuntimeError: Error in Opti::solve [OptiNode] at .../casadi/core/optistack.cpp:157:
.../casadi/core/function_internal.cpp:145: Error calling QpoasesInterface::init for 'solver_qpsol':
.../casadi/interfaces/qpoases/qpoases_interface.cpp:839: No such qpOASES::PrintLevel: 0
```

**Reply by jgillis** [reply]
Options are given here: https://github.com/casadi/casadi/blob/7f8f055024ed6d86532612f1b706195c7d41074a/casadi/interfaces/qpoases/qpoases_interface.cpp#L825
Try "none"
